yt_dlp/extractor/brightcove.py

```diff
@@ -632,7 +906,7 @@ def extract_policy_key():
 
         api_url = 'https://edge.api.brightcove.com/playback/v1/accounts/%s/%ss/%s' % (account_id, content_type, video_id)
         headers = {}
-        referrer = smuggled_data.get('referrer')
+        referrer = traverse_obj(smuggled_data, 'Referer', 'referrer', casesense=False)
```

Comment:
Since the code never actually worked with `referrer`, we don't have to keep compat
```suggestion
        referrer = smuggled_data.get('Referer')
```

Reply:
BrightcoveNewIE used to get `referrer` from smuggled_data, it was the GenericIE that was wrongly smuggling `Referer` post-regression

a couple other extractors still smuggle it as `referrer`:
https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/itv.py#L261
https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/genius.py#L67

should I edit those as well so that we are uniformly using `Referer`?

Reply:
No. That'd break `--load-info`. We could handle both cases, but that's an unnecessary change imho. Instead, it's be easier to change this back to `referrer`